src/outputs/short_form.py

```diff
@@ -0,0 +1,72 @@
+"""The main file for the Outputs module."""
+import logging
+import pandas as pd
+from datetime import datetime
+from typing import Callable, Dict, Any
+import toml
+
+import src.outputs.short_form_out as short
+import src.outputs.map_output_cols as map_o
+from src.staging.validation import load_schema, validate_data_with_schema
+from src.outputs.outputs_helpers import create_output_df
+
+
+OutputMainLogger = logging.getLogger(__name__)
+
+# Get the shortform schema
+short_form_schema = toml.load("src/outputs/output_schemas/frozen_shortform_schema.toml")
+
+def run_short_form(
```

Comment:
```suggestion
def output_short_form(
```
Might be more specific

Reply:
agree, done